image-scanning/README.adoc

```diff
@@ -0,0 +1,49 @@
+= Add Basic Image Scanning to any Cluster
+
+This project deploys a Jenkins job that will run an OpenSCAP scan of all images currently being used by `Running` pods. A report will be created for each image and archived by Jenkins into a directory structure as follows:
+
+[source,bash]
+----
+./
+  <namespace>/
+    <image-ID>/
+      report.html
+      report-arf.xml
+----
+
+The entire report can be downloaded as a ZIP for data processing/archiving purposes.
+
+== Automated Deployment with Applier
+
+This capability can be deployed to any cluster using link:https://github.com/redhat-cop/openshift-applier[OpenShift Applier].
+
+[source,bash]
+----
+cd image-scanning/
+ansible-galaxy install -r requirements.yml -p galaxy
+ansible-playbook -i inventory/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
```

Comment:
Project isn't created at this point.  Does applier create a namespace?

Reply:
Also, after running command as-is, image-sign-scan-base-1-build fails:
`Error getting repository data for rhel-7-server-ose-3.7-rpms, repository not found`

Reply:
@afcollins thanks for the review! I've added the project and removed that dependency on a particular channel. Be aware however, that your minishift will require a subscription in order to build the scanner image, as it needs to be able to install `oc` from a red hat source.